### PR TITLE
Applying changes to project just like in tdl-client-java

### PR DESCRIPTION
**Pull request by neomatrix369**
And incorporating scripts in order to start and stop dependent services

Do not merge unless its good to go.

### CODE REVIEW

**Comment by neomatrix369** [comment]
Btw there is ambiguity in the names used, i.e. actions have two different meanings, one of them has been deleted but does the one that is used under `runner_action` different in meaning?

File: src/tdl/queue/queue_based_implementation_runner.py [hunk 1]
```diff
@@ -114,11 +115,13 @@ def process_next_request_from(self, remote_broker, headers, request):
         response = self._processing_rules.get_response_for(request)
         self._audit.log(response)
 
-        client_action = response.client_action
-        self._audit.log(client_action)
+        #TODO: check again if this is correctly done, come back later to complete it
```
Comment (neomatrix369) [inline src/tdl/queue/queue_based_implementation_runner.py:118]: @julianghionoiu In python how do we want to handle these edge cases, if there are multiple messages and one of the middle ones fails due to any of the two reasons, we should terminate the message listening process

Comment (neomatrix369) [reply src/tdl/queue/queue_based_implementation_runner.py:118]: At the moment we are not handling it effectively as you can see from the below code

Comment (neomatrix369) [reply src/tdl/queue/queue_based_implementation_runner.py:118]: We still have a case of both the messages reaching the response queue

Comment (neomatrix369) [reply src/tdl/queue/queue_based_implementation_runner.py:118]: If 3 messages are sent and then 2nd one fails during processing, so recevied messages is still 2 right? And published messaged is 1?

**Comment by neomatrix369** [comment]
Same here, the coveralls check is preventing fulfilling of requirements for merging

File: src/tdl/queue/queue_based_implementation_runner.py (hunk 1 above)
Comment (julianghionoiu) [reply src/tdl/queue/queue_based_implementation_runner.py:118]: Yes, that is correct.

You can remove the comment

**Comment by julianghionoiu** [comment]
> runner_action

We will be keeping the runner action concept, this is the current action the user asks the runner to perform.

**Review by julianghionoiu — APPROVED** [review]

**Comment by julianghionoiu** [comment]
Looks good and tests are passing.

**Comment by julianghionoiu** [comment]
Merged.
tdl-client-python v0.25.0 is now available to use in the runner

**Pull request merged**